script/configs/exclude_integration_web.yaml

```diff
@@ -1,4 +1,2 @@
 # Currently missing: https://github.com/flutter/flutter/issues/82211
 - file_selector
-# Waiting on https://github.com/flutter/flutter/issues/145149
-- google_maps_flutter
```

Comment:
Yikes! is this also preventing `google_maps_flutter_web` from running on CI?

Reply:
It does:

```
============================================================
|| Not running for packages/google_maps_flutter/google_maps_flutter_web; excluded [@7:02]
============================================================
```

Reply:
I believe they were both failing the same way.

Reply:
I have a "fix" for `_web` at least:

* https://github.com/flutter/packages/pull/7269

(The root cause seems to be a bug in the `html` renderer not letting us adjust the `devicePixelRatio`)

For the web, I just stripped the DPR overrides from the tests (for now), but in cross-platform tests the DPR overrides probably make more sense.

Reply:
I think the only tests that need to be disabled are those that set the `tester.view.devicePixelRatio` AND are running with the `html` renderer (`CHANNEL=stable`). Canvaskit seems to not be affected.